### PR TITLE
feat: readme aescan.io link

### PR DESCRIPTION
**Pull request by michele-franchi**
## Description

Add aescan.io links to the Readme file.

## Checklist:

- [ ] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,12 +92,15 @@ Three different deployment stages are available to test or use æScan:
     </tr>
     <tr>
         <td>Production</td>
-        <td>https://aescan.prd.aepps.com/</td>
+        <td>
+            https://aescan.io
+            https://aescan.prd.aepps.com
```
Comment (janmichek) [inline README.md:97]: Do we need to mention `https://aescan.prd.aepps.com` ?

**Review by janmichek — APPROVED** [review]
Thanks for keeping readme updated. I have a preference not to show the other address, but no strong preference. As you wish .)

**Review by lukeromanowicz — APPROVED** [review]
Apart from what Jan has already pointed out, looks good :+1:

**Comment by michele-franchi** [comment]
@michele-franchi @janmichek agree, I removed the secondary production link.

**Pull request merged**